system-test/spanner.ts

```diff
@@ -4393,6 +4396,37 @@ describe('Spanner', () => {
       });
     });
   });
+
+  describe('Resource Based Routing', () => {
+    const database = instance.database(generateName('database'));
+    const table = database.table('ResourceBasedRoutingTable');
+
+    const schema = `
+      CREATE TABLE UserTable (
+        Key STRING(MAX) NOT NULL,
+        Name STRING(MAX)
+      ) PRIMARY KEY (Key)
+    `;
+
+    before(async () => {
+      await onPromiseOperationComplete(await database.create());
+      await onPromiseOperationComplete(await table.create(schema));
+    });
+    beforeEach(() => {
+      spanner.options.enableResourceBasedRouting = true;
+    });
+    afterEach(() => {
+      spanner.options.enableResourceBasedRouting = false;
+    });
+
+    it('should list the session while resource based routing is enabled', done => {
+      database.getSessions((err, session) => {
+        assert.ifError(err);
+        assert.equal(session!.length, 0);
```

Comment:
Why would the number of sessions be zero?

Reply:
because it's newly created database.

Reply:
Interesting. I thought we initialized the session pool with a min number of sessions when the the client is first initialized. Is this not the case?

Reply:
@olavloite explained in a separate comment that the Node.js library initializes the session pool with min=0.